### DISCUSSION TITLE
Error: uawidgets.utils - ERROR - 'Field' object is not subscriptable')

### DISCUSSION THREAD

**Discussion by derhardi02** [FreeOpcUa/python-opcua#1514] [Q&A]
Hi everyone,

I got an issue with OPC-UA Client. I am trying to connect the client with a OPC-UA Server running on a Siemens S7-1500 because I want to read data from optimized data blocks. After several warnings in my terminal which look like this: 
asyncua.common.structures104 - WARNING - renamed "UDT_Q_F-Switch to UDT_Q_F_Swicth" due to Python syntax.
I get the error "uawidgets.utils - ERROR - 'Field' object is not subscriptable')".
I am new to the whole OPC-UA topic, thats why I hope for some help

Thank you

**Comment by kwisii** [comment]
We solved this problem by deactivating unnecessary databases.
Maybe the problem was the number of variables defined in this databases.
I can imagine that the variable numbers are limited.

**Comment by AndreasHeine** [comment]
are you in the correct repo? did you mean https://github.com/FreeOpcUa/opcua-client-gui
